js/lesson-24/part1.js

```diff
@@ -0,0 +1,39 @@
+`use strict`;
+
+// введення даних та валідація
+let $startNumber;
+let $endNumber;
+
+$startNumber = +prompt(`Введіть число - початок діапазону`);
+
+if ($startNumber === `` || isNaN($startNumber)) {
+    console.log(`Не прийнято! Спробуйте ще раз.`);
+} else {
+    console.log(`Діапазон починається з`, $startNumber);
+    $endNumber = +prompt(`Введіть число - закінчення діапазону`);
+
+    if ($endNumber === `` || isNaN($endNumber)) {
+        console.log(`Не прийнято! Спробуйте ще раз.`);
+    } else {
+        console.log(`Діапазон закінчується на`, $endNumber);
+    }
+}
+
+// розрахунок
+
+let $sum = 0;
+
+if ($startNumber < $endNumber) {
+    for (let i = $startNumber; i <= $endNumber; i++) {
+        $sum++;
```

Comment:
You need to add a next number from range. Since $sum++ will increment 0 and will be $sum = 1

Reply:
Fixed